docs/topics/object-declarations.md

```diff
@@ -47,38 +60,76 @@ window.addMouseListener(object : MouseAdapter() {
 ```
 
 If a supertype has a constructor, pass appropriate constructor parameters to it.
-Multiple supertypes can be specified as a comma-delimited list after the colon:
+Multiple supertypes can be specified, separated by commas, after the colon:
 
 ```kotlin
-open class A(x: Int) {
-    public open val y: Int = x
+//sampleStart
+// Creates an open class BankAccount with a balance property
+open class BankAccount(initialBalance: Int) {
+    open val balance: Int = initialBalance
 }
 
-interface B { /*...*/ }
+// Defines an interface Transaction with an execute() method
+interface Transaction {
+    fun execute()
+}
+
+// Function to perform a special transaction on a BankAccount
+fun specialTransaction(account: BankAccount) {
+    // Creates an anonymous class inheriting from BankAccount and implementing Transaction
```

Comment:
I like this suggestion -but what do you think about using  "implement an interface" instead of "inherit from an interface" :
So it would look like this:
// Creates an anonymous object that inherits from the BankAccount class and implements the Transaction interface

Reply:
Works for me!